packages/react-core/src/components/ToggleGroup/ToggleGroup.tsx

```diff
@@ -9,6 +10,7 @@ export interface ToggleGroupProps extends React.HTMLProps<HTMLDivElement> {
   className?: string;
   /** Modifies the toggle group to include compact styling. */
   isCompact?: boolean;
+  disableAll?: boolean;
```

Comment:
Could you add a prop description here?

Reply:
Oh snap! Sorry I forgot. Sure I will do it now.